src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -267,6 +279,18 @@ public Mnemonic CreateWallet(string password, string name, string passphrase, Mn
             return mnemonic;
         }
 
+        private void CreateDefaultWallet(bool exists)
+        {
+            if (!exists)
+            {
+                var mnemonic = new Mnemonic(Wordlist.English, WordCount.Twelve);
+                this.CreateWallet(DefaultWalletPassword, DefaultWalletName, string.Empty, mnemonic);
+            }
+
+            // Load the wallet from disk after creation, to verify it and also verify that we can decrypt it.
+            this.LoadWallet(DefaultWalletPassword, DefaultWalletName);
```

Comment:
When the node starts, all the wallets (files ending in `*.wallet.json`) contained in the data folder are loaded. 
The LoadWallet method only loads wallets that weren't loaded when the node started, for example if you have a running node and you want to load a wallet you brought from another computer.
So when the node starts, the `default.wallet.json` file will be loaded automatically.
I think you should throw a `WalletException` inside the `if` block starting on line 202 if a default wallet already exists.

Reply:
I do think at least some exchanges rely on default wallet feature, which is normally not encrypted, so auto-unlock not needed for those, but since Stratis does not support unencrypted, it must be unlocked at startup to give same features.

I don't think the mnemonic is necessary to return or give to user.

Reply:
Does the stratisX wallet have a default wallet? I know at least one exchange that uses default wallet, if there are others I presume that exchanges must either modify their integrations to support Stratis in the future, or continue to use stratisX?

Reply:
All fair points, thanks for the clarification.
@sondreb can you please address my comments here https://github.com/stratisproject/StratisBitcoinFullNode/pull/2889#discussion_r244854871 about the Load methos?

Reply:
> Does the stratisX wallet have a default wallet?

It works just like Bitcoin not HD.
They shouldn't have to change their processes, normally.

Reply:
@sondreb do you want me to take this one over?

Reply:
As soon as #3017 is merged, I can finish this. It is somewhat dependent on that fix and the ability to set unlock period.

Reply:
I will improve the description of the "defaultwallet" argument. I don't think it should throw exception if it already exists, because I'd like to be able to have a single docker-compose.yml that I can run and re-run without needing any re-configuration and re-deployment. But the description should say "Loads the default wallet on startup. If not exists, will be created automatically."

Reply:
Can I do the following: Modify the GetAccount in RPC controller to retrieve the "default" wallet if it exists, otherwise use the first in the array, as before? It is going to be very weird if someone for some reason has two wallets (which one normally shouldn't do if one want to use RPC, at least not at this stage), and on startup it unlocks default, but when you perform "sendtoaddress", it loads a different wallet...

Reply:
Please check the latest code, should be fairly clear.